lib/msal-node/src/cache/Storage.ts

```diff
@@ -215,8 +216,9 @@ export class Storage extends CacheManager {
      */
     getAccessTokenCredential(accessTokenKey: string): AccessTokenEntity | null {
         const credType = CredentialEntity.getCredentialType(accessTokenKey);
-        if (credType === CredentialType.ACCESS_TOKEN) {
-            return this.getItem(accessTokenKey) as AccessTokenEntity;
+        const accessToken = this.getItem(accessTokenKey);
+        if (accessToken && credType === CredentialType.ACCESS_TOKEN) {
```

Comment:
Same q for access token and refresh token

Reply:
Answered above.

Reply:
Done.